### PR TITLE
Updated index.html

### PR DESCRIPTION
**Pull request by yashar1908**
Changed a few sentences, made no change to the code as such...only changed the formation of the sentences that were displayed

### CODE REVIEW

**Comment by RITEKROUNAK** [comment]
@Mahi-Badaya 
Please Review it Once.

File: index.html [hunk 1]
```diff
@@ -148,7 +148,7 @@ <h2 class="typed_active">Open Source ?</h2>
                     <p>
                       We encourage all the <strong>developers</strong> to come with us and join this revolution of availing 
                       technology to everyone through <strong>Open Source Contribution</strong>.
-                      We Provide you all the credits of <strong>development</strong>, So that the users can know who are <strong>Developers </strong>
+                      We provide to you, all the credits of <strong>development</strong>, So that the users can know who are  the<strong> Developers </strong>
```
Comment (Mahi-Badaya) [inline index.html:151]: -We provide you all the credits of
this will make more sense

File: index.html [hunk 2]
```diff
@@ -284,7 +284,7 @@ <h4 class="heading"><i class="fas fa-mobile"></i>YOUR APPS WOULD BE PUSHED TO PL
                   </div> 
   
                   <div class="paragraph-wrapper">
-                      <p class="paragraph">Hmmm, it seems an another cool reason.. isn’t it? We will push your apps on <strong>Google PlayStore</strong>  	and in the future, maybe <strong>AppStore</strong> as well :) So that you can only focus on developing the app 	and leaving all monetary and maintenance burdens to us<strong>(WE DO LOVE BURDENS).</strong></p>
+                      <p class="paragraph">We will push your apps on <strong>Google PlayStore</strong>  	and in the future, maybe <strong>AppStore</strong> as well :) So that you can only focus on developing the app 	and leaving all monetary and maintenance burdens to us</p>
```
Comment (Mahi-Badaya) [inline index.html:287]: Hmmm, it seems another cool reason.. isn’t it?

We can keep this line like this.

File: index.html [hunk 3]
```diff
@@ -302,7 +302,7 @@ <h4 class="heading"><i class="fas fa-link"></i> CONNECT CONNECT CONNECT</h4>
                   </div> 
   
                   <div class="paragraph-wrapper">
-                      <p class="paragraph">Guess everyone knows how much important connections are! And what can be a better way 	to connect by working together on some project. Open source is meant <strong>FOR EVERYONE BY 	EVERYONE.</strong> Innovate Ideas, Develop and Deploy while working with a great team motivated 	to bring change in real world through Open-Source Contributions.</p>
+                      <p class="paragraph">Having connections is really important, and what better way to connect than by working together on a project. Open source is meant <strong>FOR EVERYONE BY 	EVERYONE.</strong> Innovate Ideas, Develop and Deploy while working with a great team motivated 	to bring change in real world through Open-Source Contributions.</p>
```
Comment (Mahi-Badaya) [inline index.html:305]: No changes are required in the original content.

File: index.html [hunk 4]
```diff
@@ -337,7 +337,7 @@ <h4 class="heading"><i class="fas fa-certificate"></i> CERTIFICATES</h4>
                 </div> 
 
                 <div class="paragraph-wrapper">
-                    <p class="paragraph">Yes, we know you must be thinking it doesn’t matters much but wait, there is a <strong>catch!</strong>! We provide 	Certificates with <strong>Certificate IDs</strong>, which acts as proof to show your recruiter your contribution 	to the open-source projects...annnddD companies love candidates who make Open Source 	Contributions. So your Resume is undoubtedly going to shine in front of any recruiter.</p>
+                    <p class="paragraph">Yes, we know you must be thinking it doesn’t matter much but wait, there is a <strong>catch!</strong>! We provide 	Certificates with <strong>Certificate IDs</strong>, which acts as proof to show your recruiter your contribution 	to the open-source projects, and companies love candidates who make Open Source 	Contributions. So your Resume is undoubtedly going to shine in front of any recruiter.</p>
```
Comment (Mahi-Badaya) [inline index.html:340]: only change matters to matter. else was okay previously.

File: index.html [hunk 5]
```diff
@@ -262,8 +262,8 @@ <h4 class="heading"><i class="fas fa-globe"></i> FREE HOSTING AND SERVER</h4>
                   </div> 
   
                   <div class="paragraph-wrapper">
-                      <p class="paragraph">Oops seems we started it from Bang! But yea, you heard it right.
-                        All your projects would be hosted for free at <strong>NO EXTRA COST!</strong> On BlackTimber-Labs Physical 	Servers. Forget about those heavy charges by AWS , AZURE, GCLOUD etc to host your 	projects.
+                      <p class="paragraph">
+                        All your projects would be hosted for free at <strong>NO EXTRA COST!</strong> On BlackTimber-Labs' Physical 	Servers. Forget about those heavy charges by AWS , AZURE, GCLOUD etc to host your 	projects.
```
Comment (Mahi-Badaya) [inline index.html:266]: Don't remove the line "Oops seems we started it from Bang! But yea, you heard it right." it's fine

**Review by Mahi-Badaya — CHANGES_REQUESTED** [review]
Please make the mentioned changes

**Review by Mahi-Badaya — APPROVED** [review]
It's okay now.

**Comment by yashar1908** [comment]
But I didn't change anything 😭😂

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**